webpack.config.js

```diff
@@ -66,13 +70,11 @@ let config = {
 // *********************************
 
 if (process.env.NODE_ENV === "production") {
-	config.plugins.push(new webpack.optimize.DedupePlugin());
 	config.plugins.push(new webpack.optimize.UglifyJsPlugin({
-		comments: false,
-		compress: {
-			warnings: false
-		}
+		comments: false
 	}));
+} else {
+	console.log("Building in development mode, bundles will not be minified.");
```

Comment:
Indeed but saying this is not production more accurate as we also have test env, etc.. Oh well, nitpick.